src/Dapr.Actors.AspNetCore/ActorsEndpointRouteBuilderExtensions.cs

```diff
@@ -189,6 +190,7 @@ private static IEndpointConventionBuilder MapActorHealthChecks(this IEndpointRou
     {
         var responseHeader = new ActorResponseMessageHeader();
         responseHeader.AddHeader("HasRemoteException", Array.Empty<byte>());
+        responseHeader.AddHeader("RemoteMethodException", Encoding.ASCII.GetBytes(GetExceptionInfo(ex)));
```

Comment:
This is just a byte conversion, you didn't appear to do a base64 encoding first.